### ISSUE TITLE
Add in-wallet block explorer

### ISSUE THREAD

**Issue by nopara73** [zkSNACKs/Meta#19]
People can lose their privacy by checking their transactions in a block explorer over the clearnet. Wasabi could help mitigate this risk by adding an in-wallet block explorer query over Tor.

**Comment by Transisto** [comment]
I agree this is very important, ie. Can't even see receiver and sender of transaction in history.

Blockstream.info block explorer has tor endpoints and an API

https://github.com/Blockstream/esplora/blob/master/API.md

**Comment by nopara73** [comment]
A few weeks ago I researched this topic (more accurately how to show html pages in software) and we won't be able to do this anytime soon unless we use an external libraries, which is a no-go.  

https://github.com/AvaloniaUI/Avalonia/issues/1493

**Comment by MaxHillebrand** [comment]
This is related to the following issues
https://github.com/zkSNACKs/WalletWasabi/issues/2018
https://github.com/zkSNACKs/WalletWasabi/issues/817
https://github.com/zkSNACKs/WalletWasabi/issues/612
